packages/discovery-provider/src/queries/get_attestation.py

```diff
@@ -157,6 +161,17 @@ def get_attestation(
     if disbursement:
         raise AttestationError(ALREADY_DISBURSED)
 
+    if challenge.weekly_pool:
+        now = datetime.now(pytz.timezone("US/Pacific"))
+        monday_before_9am = now.weekday() == 0 and now.hour < 9
+        prev_monday = now - timedelta(days=7 if monday_before_9am else now.weekday())
+        prev_monday = prev_monday.replace(hour=9, minute=0, second=0, microsecond=0)
+        disbursed_amount = get_disbursed_challenges_amount(
+            session, challenge.id, prev_monday
+        )
+        if disbursed_amount + user_challenge.amount > challenge.weekly_pool:
```

Comment:
1. Might help to add a comment above briefly describing the logic in this bit (We want all challenges disbursed since the most recent Monday at 9am?)
2. Any reason to store any of this in config vs. hardcoded? (would we want the ability to quickly adjust the disbursement window?)
3. Worth it to make a util function instead? (`get_weekly_pool_window_start` or something like that)